### PR TITLE
[RBAC] Add RBAC object tests

### PR DESCRIPTION
**Pull request by bmclaughlin**
Adds object based tests for RBAC.

Issue: AAH-1850

### CODE REVIEW

**Review by jctanner — APPROVED** [review]

**Comment by jctanner** [comment]
I'm guessing this failure is related to rebase?

```
+ django-admin collectstatic
Traceback (most recent call last):
  File "/venv/bin/django-admin", line 8, in <module>
    sys.exit(execute_from_command_line())
  File "/venv/lib64/python3.8/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/venv/lib64/python3.8/site-packages/django/core/management/__init__.py", line 395, in execute
    django.setup()
  File "/venv/lib64/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/venv/lib64/python3.8/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/venv/lib64/python3.8/site-packages/django/apps/config.py", line 301, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib64/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/app/galaxy_ng/app/models/__init__.py", line 13, in <module>
    from .synclist import (
  File "/app/galaxy_ng/app/models/synclist.py", line 4, in <module>
    from pulpcore.plugin.models import AutoDeleteObjPermsMixin
ImportError: cannot import name 'AutoDeleteObjPermsMixin' from 'pulpcore.plugin.models' (/venv/lib64/python3.8/site-packages/pulpcore/plugin/models/__init__.py)
The command '/bin/sh -c set -ex;     mkdir --mode=2775 -p /etc/pulp/certs/ &&     if [[ "${LOCK_REQUIREMENTS}" -eq "1" ]]; then     PULP_CONTENT_ORIGIN=x django-admin collectstatic; fi' returned a non-zero code: 1
Service '_base' failed to build : Build failed
make: *** [Makefile:83: docker/build] Error 1
```

**Comment by bmclaughlin** [comment]
@jctanner That's correct.  Rebasing now that https://github.com/ansible/galaxy_ng/pull/1392 is merged.

**Comment by bmclaughlin** [comment]
CI errors fixed in https://github.com/ansible/galaxy_ng/pull/1395

File: galaxy_ng/tests/integration/api/test_rbac_roles.py [hunk 1]
```diff
@@ -340,6 +329,71 @@ def test_global_role_actions(role):
...
+    registry = ReusableContainerRegistry(gen_string())
+    registry_pk = registry.get_registry()["pk"]
+
+    users_and_groups = {}
+    col_groups = []
+    ee_groups = []
+
+    # Create user/group for each role
+    # Populate collection/ee groups for object assignment
+    for role in ROLES_TO_TEST:
```
Comment (newswangerd) [inline galaxy_ng/tests/integration/api/test_rbac_roles.py:344]: We don't need to test all the roles. The UI only lets you assign object permissions for roles that only have permissions on the object. Take a look at the UI for namespace and EE owners and see which roles are allowed, and let's stick to just testing those.

**Review by newswangerd — CHANGES_REQUESTED** [review]
Looks good! Couple of small changes.

**Review by newswangerd — APPROVED** [review]

**Pull request merged**